apps/bfd-pipeline/bfd-pipeline-rda-bridge/src/main/java/gov/cms/bfd/pipeline/bridge/etl/FissTransformer.java

```diff
@@ -202,7 +202,7 @@ FissClaimChange transformNewClaim(
             RecordSource.newBuilder()
```

Comment:
There is nothing wrong with this line, but this PR looks like it's _just_ about ready to merge. Just holding it until 1292 lands.

Reply:
@mjburling did you mean PR #1294?  This PR is 1292.

Reply:
> @mjburling did you mean PR #1294? This PR is 1292.

Oh, dear. Yes, indeed. Now that this is through, I think we're all set. Sorry for the delay, @ericbuckley.